packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://github.com/rancher/aks-operator/releases/download/v1.0.3/rancher-aks-operator-crd-1.0.3.tgz
```

Comment:
We'd need to change this to `v1.0.4-rc1`, https://github.com/rancher/charts/pull/1625 for reference.